### PR TITLE
tg rebase work

### PR DESCRIPTION
**Pull request by SuperNovaa41**
tg-rebase work (NEW pr because i goofe dhte old one)

### CODE REVIEW

File: voidcrew/_maps/RandomRuins/SandRuins/whitesands_surface_saltmine [hunk 1]
```diff
@@ -0,0 +1,1237 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
```
Comment (Dennok) [inline voidcrew/_maps/RandomRuins/SandRuins/whitesands_surface_saltmine:1]: Lost .dmm ?

File: voidcrew/code/datums/components/crafting/recipes.dm [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+				/obj/item/gibtonite = 1)
+	tool_behaviors = list(TOOL_SCREWDRIVER)
+	category = CAT_MISC
+
+/datum/crafting_recipe/spickaxe
+	name = "Improvised Pickaxe"
+	reqs = list(
+		   /obj/item/crowbar = 1,
+		   /obj/item/knife/kitchen = 1,
+		   /obj/item/restraints/handcuffs/cable/sinew = 1)
```
Comment (Dennok) [inline voidcrew/code/datums/components/crafting/recipes.dm:24]: Don't sure that it match here, but for TO DO at least.
Please, make it use just handcuffs/cable, so who have just autolathe can create pickaxe.

File: voidcrew/code/modules/mining/equipment/regenerative_core.dm [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+				to_chat(user, span_notice("[src] is useless on the dead."))
+				return
+			if(H != user)
+				H.visible_message(span_notice("[user] forces [H] to apply [src]... Cancer like crystals grow on and reinforce [H.p_them()]!"))
+				SSblackbox.record_feedback("nested tally", "hivelord_core", 1, list("[type]", "used", "other"))
+			else
+				to_chat(user, span_notice("You start to apply [src] on yourself. Cancer like crystals hold you together and add something to you to keep yourself moving, but for how long?"))
+				SSblackbox.record_feedback("nested tally", "hivelord_core", 1, list("[type]", "used", "self"))
+			H.apply_status_effect(/datum/status_effect/regenerative_core)
+			H.reagents.add_reagent(/datum/reagent/determination, 4)
```
Comment (Dennok) [inline voidcrew/code/modules/mining/equipment/regenerative_core.dm:40]: Here reagent too.

File: voidcrew/code/modules/mob/living/carbon/human/species_types/IPC.dm [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+	//species_gibs = "robotic"
+	attack_sound = 'sound/items/trayhit1.ogg'
+	/// VC SPECIES TODO
+	//allow_numbers_in_name = TRUE
+	deathsound = "sound/voice/borg_deathsound.ogg"
+	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_MAGIC | MIRROR_PRIDE | ERT_SPAWN | RACE_SWAP | SLIME_EXTRACT
+	/// VC SPECIES TODO
+	//loreblurb = "Integrated Positronic Chassis or \"IPC\" for short, are synthetic lifeforms composed of an Artificial \
+	Intelligence program encased in a bipedal robotic shell. They are fragile, allergic to EMPs, and the butt of endless toaster jokes. \
+	Just as easy to repair as they are to destroy, they might just get their last laugh in as you're choking on neurotoxins. Beep Boop."
```
Comment (Dennok) [inline voidcrew/code/modules/mob/living/carbon/human/species_types/IPC.dm]: Need more commenting?

**Comment by SuperNovaa41** [comment]
I would really wait for review beacuse this is far from done

**Pull request merged**